network/application/mqtt_client.c

```diff
@@ -0,0 +1,293 @@
+/***************************************************************************//**
+ *   @file   mqtt_client.c
```

Comment:
commit message must be 75 characters long.

Reply:
done